### PR TITLE
Add GYM-5049 content

### PR DESCRIPTION
**Pull request by jgagne**
## What this PR does:

- Adds new Take 5 GYM-5049
- Updates home page hero to feature GYM-5049
- Adds GYM-5049 content: front matter, meta, transcript, and posters and OG artwork
- Updates project file source logic
- Updates Take 5 home page hero styling
- Updates test header and footer content (for local development)
- Updates GYM-5048 content; removes errant space

### Preview: GYM-5049 Content

- https://deploy-preview-619--thegymcms.netlify.app/courses/take5/gym-5049

### Preview: Take 5 Home Page Hero

gym-5049-home-page-hero

### Take 5 Status

- https://deploy-preview-619--thegymcms.netlify.app//tests/take5-content/

### CODE REVIEW

**Review by rediris — DISMISSED** [review]
Looks good to me.

File: _data/hero.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
-current_promo: hero-design-systems.html
-auth_promo: hero-design-systems.html
+current_promo: take5/hero-take5-featured.html
```
Comment (rediris) [inline _data/hero.yml:1]: No need to take action now, but I wonder if there's some benefit to moving all the hero includes into a heros folder or something - that main includes directory is a little long in the tooth, and most of the files therein are archived heros.

Comment (jgagne) [reply _data/hero.yml:1]: I'd be cool with that!

Comment (rediris) [reply _data/hero.yml:1]: Cool, let's save it for the future!

Comment (andrewpmiller) [reply _data/hero.yml:1]: Added to the rainy day ☔ triage

**Review by andrewpmiller — APPROVED** [review]
## VERRRRRRY taste!

File: _data/hero.yml (hunk 1 above)
Comment (rediris) [reply _data/hero.yml:1]: Thank you!

**Pull request merged**